corehq/ex-submodules/casexml/apps/phone/migrations/0004_auto_20191021_1308.py

```diff
@@ -0,0 +1,36 @@
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('phone', '0003_auto_20190405_1752'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='synclogsql',
+            name='error_date',
+            field=models.DateTimeField(blank=True, null=True),
+        ),
+        migrations.AlterField(
+            model_name='synclogsql',
+            name='log_format',
+            field=models.CharField(choices=[('simplified', 'simplified'), ('livequery', 'livequery')], max_length=10),
+        ),
+        migrations.AlterField(
+            model_name='synclogsql',
+            name='user_id',
```

Comment:
I think this block removes the `user_id` index. Because this is before the migration where you add the composite index, any queries with user_id between this and when the composite index is added (unknown how long that would take on ICDS since its concurrently), are going to be much slower

Reply:
We don't currently query by `user_id` so this index isn't used.